lib/spack/spack/cmd/common/arguments.py

```diff
@@ -112,7 +113,7 @@ def default(self):
         # This default is coded as a property so that look-up
         # of this value is done only on demand
         return min(spack.config.get('config:build_jobs', 16),
-                   multiprocessing.cpu_count())
+                   fork_context.cpu_count())
```

Comment:
I don't think calls to `multiprocessing` other than `multiprocessing.Process` need to be controlled by `fork_context`.